modules/aws/aws.go

```diff
@@ -182,31 +179,36 @@ func (r *run) getLdaper(shortdn string) (uid string, hasPGPKey bool, err error)
 	if r.Conf.NotifyUsers {
 		// make sure we can find a PGP public key for the user to encrypt the notification
 		// if no pubkey is found, log an error and set the user's entry to False
-		_, err = r.Conf.LdapCli.GetUserPGPKey(shortdn)
+		_, pgpErr := r.Conf.LdapCli.GetUserPGPKey(shortdn)
 		if err != nil {
 			r.debug("aws %q: no pgp public key could be found for %s: %v",
-				r.p.AccountName, shortdn, err)
-			hasPGPKey = false
+				r.p.AccountName, shortdn, pgpErr)
 		}
+		hasPGPKey = true
 	}
 	return
 }
 
 func (r *run) getIamers() (igm map[string]bool) {
 	igm = make(map[string]bool)
 	for _, group := range r.p.IamGroups {
-		resp, err := r.cli.GetGroup(&iam.GetGroupInput{
+		err := r.cli.GetGroupPages(&iam.GetGroupInput{
 			GroupName: aws.String(group),
+		}, func(page *iam.GetGroupOutput, lastpage bool) bool {
+			for _, user := range page.Users {
+				iamuser := strings.Replace(awsutil.Prettify(user.UserName), `"`, ``, -1)
+				igm[iamuser] = true
+			}
+			if lastpage {
+				return false
+			}
+			return true
```

Comment:
Is this a different commit? It's not clear to me how it relates to the pgp issue.